docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+services:
```

Comment:
Added this since I wanted to check if I didnt break anything of egregious. Figured it might be useful for others in the future.